src/index.js

```diff
@@ -156,12 +158,19 @@ li {
 // Fetch metadata (used on all pages)
 store.dispatch(getMetadata());
 
+ReactGA.initialize('UA-55757642-1');
+ReactGA.pageview(window.location.pathname + window.location.search);
+const history = createHistory();
+history.listen((location) => {
+  ReactGA.pageview(location.pathname);
+});
+
 const rootElement = document.getElementById('root');
 const app = (
   <Provider store={store}>
-    <BrowserRouter>
+    <Router history={history}>
```

Comment:
does BrowserRouter not work here?

Reply:
No.  BrowserRouter does not take history.